swift/ql/lib/codeql/swift/elements/pattern/NamedPattern.qll

```diff
@@ -1,5 +1,40 @@
 private import codeql.swift.generated.pattern.NamedPattern
+private import codeql.swift.elements.pattern.BindingPattern
+private import codeql.swift.elements.pattern.EnumElementPattern
+private import codeql.swift.elements.pattern.IsPattern
+private import codeql.swift.elements.pattern.OptionalSomePattern
+private import codeql.swift.elements.pattern.ParenPattern
+private import codeql.swift.elements.pattern.TuplePattern
+private import codeql.swift.elements.pattern.TypedPattern
+private import codeql.swift.elements.decl.VarDecl
+private import codeql.swift.elements.stmt.LabeledConditionalStmt
+private import codeql.swift.elements.stmt.ConditionElement
 
 class NamedPattern extends Generated::NamedPattern {
```

Comment:
Could you add a QLDoc to this class to describe what a `NamedPattern` is? Ideally with Swift examples showing cases where the is a `VarDecl` and where the isn't a `VarDecl`.